cli/src/browser.rs

```diff
@@ -46,8 +46,8 @@ async fn start_inner(chain_spec: String, log_level: String) -> Result<Client, Bo
 	info!("👤 Role: {}", config.display_role());
 
 	// Create the service. This is the most heavy initialization step.
-	let (task_manager, rpc_handlers) = service::kusama_new_light(config)
-		.map_err(|e| format!("{:?}", e))?;
+	let builder = service::NodeBuilder::new(config);
+	let (task_manager, rpc_handlers) = builder.build_light().map_err(|e| format!("{:?}", e))?;
```

Comment:
I changed this. @expenses maybe you know if this is correct?

Before: it uses only kusama
After: it uses kusama if the chain spec is kusama, westend if westend **AND fallback to polkadot if it can't say what chain spec it is**

It sounds reasonable but maybe there is a reason why we forced kusama in the browser?

Reply:
No, there wasn't a reason for doing that as far as I know.

Reply:
So what you've done is probably an improvement.

Reply:
Purrfect

Reply:
:smile_cat: